modules/ROOT/pages/asset-usage-and-engagement-metrics-dashboard.adoc

```diff
@@ -0,0 +1,78 @@
+= Asset Usage and Engagement Metrics Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Exchange administrators can view usage and engagement metrics for API specification fragments and REST APIs using the dashboard. The dashboard provides a holistic view of asset usage and consumption within the organization. The usage metrics measure how many times an asset is downloaded from Exchange, imported from API Designer, imported from Anypoint Studio, and the number of dependents and contracts. The asset engagement score is a composite metric that administrators and asset owners can use to measure the usage of the assets in Exchange. The results for this engagement are the different consumption actions that end-users make across different points of need within Exchange. The engagement score reflects the usage of the asset in the context of the usage for API specification fragments and REST APIs in the customer root organization and business groups. 
```

Comment:
```suggestion
Exchange administrators can use the dashboard to view usage and engagement metrics for API specification fragments and REST APIs. The dashboard provides a holistic view of asset usage and consumption within the organization. The usage metrics measure how many times an asset is downloaded from Exchange, imported from API Designer and Anypoint Studio, and the number of dependents and contracts. The asset engagement score is a composite metric that administrators and asset owners can use to measure the usage of the assets in Exchange. The results of this engagement are the different consumption actions that end-users make across different points of need within Exchange. The engagement score reflects the usage of the asset in the context of the usage of API specification fragments and REST APIs in the customer's root organization and business groups. 
```

Reply:
Do you think this sentence is necessary?
"The results of this engagement are the different consumption actions that end-users make across different points of need within Exchange."

IMO it's not but I will leave that up to you.